Gopkg.lock

```diff
@@ -51,23 +65,52 @@
   revision = "07f5e79768022f9a3265235f0db4ac8c3f675fec"
   version = "v1.3.1"
 
+[[projects]]
+  branch = "master"
+  name = "github.com/grpc-ecosystem/grpc-opentracing"
+  packages = ["go/otgrpc"]
+  revision = "0e7658f8ee99ee5aa683e2a032b8880091b7a055"
+
 [[projects]]
   name = "github.com/infobloxopen/atlas-app-toolkit"
   packages = [
     "gw",
     "health",
+    "mw/auth",
     "op",
     "op/gorm"
   ]
   revision = "2fde752676ad546628535aedf8106b2579624bc8"
   version = "v0.2.0"
 
+[[projects]]
```

Comment:
Rather than pulling in this and all of themis, shouldn't we be able to just regenerate the PEP client code from the themis .proto file? Alternatively, can't we just pull in that one part of the Themis package (the client)?

Reply:
> Alternatively, can't we just pull in that one part of the Themis package (the client)?

I chose to use the PEP client inside the authorization interceptor. The PEP client uses the `pdp` and `pdp-service` packages from Themis (see [here](https://github.com/infobloxopen/themis/blob/master/pep/unmarshal.go#L13)), so I have to vendor both of these libraries. Fortunately, `dep` recognizes that I don't need every package in Themis, so it ends up vendoring only the `pep`, `pdp`, and `pdp-service` libraries (as well as the dependencies for each of these).

```
[[projects]]
  branch = "master"
  name = "github.com/infobloxopen/themis"
  packages = [
    "pdp",
    "pdp-service",
    "pep"
  ]
  revision = "b96549da41c733dfbfdfd09f0d558d3ba14b030f"
```

Reply:
John's is a good question, which I'm not certain to the answer... should we be importing the `.proto` file and doing our own codegen on it for these things, or just importing them directly from the source?

While the former seems like a nice approach, the latter has better support for actually importing the dependency in a buildchain way. The only way I know to do the former is to actually copy the proto into our project and codegen off of it, which feels a little too loosely coupled for a service we directly depend on. I dunno... other opinions?